templates/procuration/proposal.html.twig

```diff
@@ -262,14 +262,33 @@
                         <input type="hidden" name="g-recaptcha-response" value="fake" />
                     {% endif %}
 
+                    <p class="b__nudge--top-large text--small text--gray text--justify" id="procuration_legal_notices">
+                        Les informations marquées d’un astérisque sont obligatoires, l’absence de réponse de ces champs
+                        ne permettra pas à LaREM de traiter votre demande. Les données recueillies dans ce formulaire sont
+                        traitées par LaREM dans le cadre de la plateforme de procuration, afin (i) d’identifier des
+                        personnes susceptibles d’être mandataires en vue d’obtenir la procuration d’un tiers dans le cadre
+                        des élections européennes 2019, (ii) d’identifier des personnes susceptibles d’être mandants en vue
+                        d’octroyer une procuration à un tiers dans le cadre des élections européennes 2019, et (iii) de
+                        mettre en relation des mandataires et des mandants afin que les premiers puissent recevoir
+                        la procuration des seconds dans le cadre des élections européennes 2019.<br/><br/>
+
+                        Si vous y consentez, ces informations peuvent également être utilisées par LaREM afin de vous
+                        contacter pour la prochaine échéance électorale, à savoir les élections municipales 2019.<br/><br/>
+
+                        Ces informations sont nécessaires pour vous permettre de devenir mandant et peuvent être
+                        communiquées aux services administratifs habilités par LaREM et chargés de la gestion des
+                        procurations, comme nos responsables procuration, les mairies et bureaux de vote.<br/><br/>
+
+                        Conformément à la règlementation en vigueur, vous disposez d’un droit d’opposition et d’un droit à la limitation du traitement de données vous concernant, ainsi que d’un droit d’accès, de rectification, de portabilité et d’effacement de vos données. Vous disposez également de la faculté de donner des directives sur le sort de vos données après votre décès. Vous pouvez exercer vos droits en nous adressant votre demande accompagnée d’une copie de votre pièce d’identité à l’adresse postale ou électronique suivante : La République En Marche, 63 rue Sainte-Anne, 75002 Paris, France ou mes-donnees@en-marche.fr.Pour plus d’information vous pouvez consulter la Politique de protection des données de LaREM <a href="https://en-marche.fr/politique-protection-donnees" target="_blank" rel="noopener noreferrer" class="link--no-decor">https://en-marche.fr/politique-protection-donnees</a>.
```

Comment:
A mailto link is not needed for the `mes-donnees@en-marche.fr` email address ?

Missing whitespace here : ` mes-donnees@en-marche.fr.Pour plus d’information`